css/css-backgrounds/border-image-shorthand-002.htm

```diff
@@ -20,16 +26,13 @@
                 border: 20px double red;
                 border-image-repeat: stretch;
                 border-image-slice: 40% 15% 20% 5%;
-                border-image-source: url("../support/9grid40-30-20-10-lime.png");
+                border-image-source: url("support/9grid40-30-20-10-lime.png");
```

Comment:
This test should probably use the red version.

Reply:
Yes.